Improve link to GitHub example (#7204)
jenkins-infra/jenkins.io: content/doc/developer/publishing/wiki-page.adoc

. Modify the URL documentation page reference in the project file so that it points to GitHub (link:/doc/developer/publishing/documentation/#referencing-the-documentation-page-from-the-project-file[documentation]).
. Commit changes, push them to your fork and create a pull request against the repository.

NOTE: After migration, the continued existence of out-of-date wiki pages causes several problems, such as duplication, confusion, and unclear migration progress.
This persistence makes it harder to identify which pages have migrated and those waiting for migration.
If you cannot identify migrated pages, this creates a major technical obstacle in content migration from the wiki.

> When migrating documentation, the out-of-date wiki page must be removed and have a redirect configured to point to the migrated version.

Reviewer: ```suggestion
When migrating documentation, the out-of-date wiki page must have a redirect configured to point to the migrated version.
```

> NOTE: After migration, the continued existence of out-of-date wiki pages causes a number of problems, such as duplication, confusion, and unclear migration progress. 
> This persistence not only makes the process more difficult overall, but it also makes it harder to identify which pages have already been migrated and which are still waiting. 
> This is a major technical obstacle in the way of content migration from the wiki. 
> When migrating documentation, after the material has been properly moved, the out-of-date wiki page must be removed and a redirect must be configured to the migrated version. 

Reviewer 2: ```suggestion
NOTE: After migration, the continued existence of out-of-date wiki pages can cause several problems, such as duplication, confusion, and unclear migration progress. 
This persistence makes the process more difficult overall and makes it harder to identify which pages have already migrated and those still waiting for migration. 
This is a major technical obstacle for content migration from the wiki. 
When migrating documentation, after the material has been properly moved, you must remove the out-of-date wiki page and configure a redirect to the migrated version.  
```
I have provided an updated suggestion, as this is a bit better than what is currently written/what I had suggested previously.

> NOTE: After migration, the continued existence of out-of-date wiki pages causes a number of problems, such as duplication, confusion, and unclear migration progress. 
> This persistence not only makes the process more difficult overall, but it also makes it harder to identify which pages have already been migrated and which are still waiting. 
> This is a major technical obstacle in the way of content migration from the wiki. 
> When migrating documentation, after the material has been properly moved, the out-of-date wiki page must be removed and a redirect must be configured to the migrated version. 
> 

Reviewer 2: ```suggestion
NOTE: After migration, the continued existence of out-of-date wiki pages causes several problems, such as duplication, confusion, and unclear migration progress.
This persistence makes it harder to identify which pages have migrated and those waiting for migration.
If you cannot identify migrated pages, this creates a major technical obstacle in content migration from the wiki.
When migrating documentation, the out-of-date wiki page must be removed and have a redirect configured to point to the migrated version.

```
I have provided a suggestion to rework the content so that it reads a bit more smoothly, reduces unnecessary words/phrases, and uses active voice. I've also removed any trailing spaces left at the end of lines.

. Modify the URL documentation page reference in the project file so that it points to GitHub (link:/doc/developer/publishing/documentation/#referencing-the-documentation-page-from-the-project-file[documentation]).
. Commit changes, push them to your fork and create a pull request against the repository.

> Note:- There is a major technical obstacle in the way of content migration from the wiki. This difficulty stems from the requirement that, when the material has been properly moved, the out-of-date wiki page be removed and a redirection mechanism to the migrated version be set up. After migration, the continued existence of out-of-date wiki pages causes a number of problems, such as duplication, confusion, and unclear migration progress. This persistence not only makes the process more difficult overall, but it also makes it more difficult to identify which material has already been migrated and which is still waiting, which makes the work at hand much more difficult.

Reviewer 2: ```suggestion
NOTE: After migration, the continued existence of out-of-date wiki pages causes a number of problems, such as duplication, confusion, and unclear migration progress. This persistence not only makes the process more difficult overall, but it also makes it harder to identify which pages have already been migrated and which are still waiting. This is a major technical obstacle in the way of content migration from the wiki. When migrating documentation, after the material has been properly moved, the out-of-date wiki page must be removed and a redirect must be configured to the migrated version. 
```
I have reworded some of the content, and moved it accordingly to try and make it read more clearly. I have also adjusted the `NOTE:` macro so that it is implemented as a note on the page, thus giving it a different visual/making it stand out more. There were also a few instances where a singular version of a word was used instead of the plural.

Author: Along with this I have made new line for each sentence also
